docs/standard/assembly/create-use-command-line.md

```diff
@@ -0,0 +1,182 @@
+---
```

Comment:
This file should simply be deleted. It doesn't actually show how to create and use assemblies (a DLL) from the command line, and has very few page views (88 for the last 30 days). Revising it both to do what it says and to cover .NET Core will require far more work than is justified by the number of page views.